controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/JobRunner.java

```diff
@@ -32,29 +37,37 @@ public class JobRunner extends ControllerMaintainer {
     private final JobController jobs;
     private final ExecutorService executors;
     private final StepRunner runner;
+    private final Metrics metrics;
 
     public JobRunner(Controller controller, Duration duration) {
-        this(controller, duration, Executors.newFixedThreadPool(32, new DaemonThreadFactory("job-runner-")), new InternalStepRunner(controller));
+        this(controller, duration, Executors.newFixedThreadPool(32, new DaemonThreadFactory("job-runner-")),
+             new InternalStepRunner(controller));
     }
 
     public JobRunner(Controller controller, Duration duration, ExecutorService executors, StepRunner runner) {
+        this(controller, duration, executors, runner, new Metrics(controller.metric(), Duration.ofMillis(100)));
```

Comment:
Metrics gathering is done every 1 minute so the interval is typically set just shy of 60 seconds.  Why 100 ms?

Reply:
Metric.set has count, last, max, min, and sum.